Makefile

```diff
@@ -1,40 +1,41 @@
-# You may not be able to build with a modern Xcode, since we still support 10.6.
-# So, if we have Xcode 4.6 around, we should use it.
+# For the building the driver for MacOS 10.11+, the preferred
+# XCode version is 7.3.1.
```

Comment:
Why 7.3.1?  This built okay with the latest Xcode on my Mac, anyway.

Reply:
AFAIK, you need to have OS X 10.11 SDK (not a later one) in order for compiled driver to run on 10.11. The latest XCode to have this is 7.3.1, so that's what I'm using.
Do you suggest a different approach? I know you can copy the older SDK to the newer XCode. It is worth the trouble?

Reply:
Huh, yeah, true enough.  I wonder why the build did not error out when I built it.  Usually if 10.11 is specified in the xcodeproj file, and the 10.11 SDK does not exist, then `xcodebuild` will just puke instead.  Ok, I'll install 7.3.1 on my system.

Reply:
You can use [xcode-legacy](https://github.com/devernay/xcodelegacy) to add older SDKs and compilers to your existing Xcode installation.

Reply:
@d235j Thanks!  The fact that people need to keep producing that kind of stuff is kind of surreal.

That page also suggested a way to tell `xcodebuild` to pick up the SDK version you want: `$(XCODEBUILD) MACOSX_DEPLOYMENT_TARGET=$(OSX_VER) SDKROOT=macosx$(OSX_VER) -project HoRNDIS.xcodeproj`.  And sure enough, when I do that with `OSX_VER=10.11` on a modern Xcode, Xcode chokes.  So now I have 7.3.1 installed (to be extra certain) and now have a successful build.

Reply:
Sorry, my fault. Before my change, the HoRNDIS project set the SDK version to 10.6. I should've set it to 10.11, but instead kept it as "use the latest SDK" - hence the confusion.

I would actually suggest setting it in the project - like before - instead of the makefile.